pages/linux/pkgadd.md

```diff
@@ -0,0 +1,15 @@
+# pkgadd
+
+> Add a package to a CRUX system
+
+- install a local software package
+
+`pkgadd {{package-name}}`
+
+- update a already installed package from a local package
+
+`pkgadd -u {{package-name}}`
+
+- force installation of a package
```

Comment:
I don't mind keeping it, it doesn't clutter the page too much yet. As long as it's somewhat useful, your judgement.